docs/source/routing/performance/caching/response-caching/quickstart.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Response caching for the GraphOS Router
+subtitle: Configure Redis-backed caching for subgraph responses
+description: Response caching for GraphOS Router with GraphOS Enterprise. Cache and reuse individual entities and root fields across queries.
+minVersion: Router v2.8.0
+releaseStage: preview
+---
+
+<PlanRequired plans={["Free", "Developer", "Standard", "Enterprise"]}>
+
+Rate limits apply on the Free plan.
+Performance pricing applies on Developer and Standard plans.
+Developer and Standard plans require Router v2.6.0 or later.
+
+</PlanRequired>
+
+Learn how the GraphOS Router can cache subgraph query responses using Redis to improve your query latency for entities in the supergraph.
+
+## Quickstart
+
+Follow this guide to enable and add a minimal configuration for response caching in the GraphOS Router.
+
+### Prerequisites
+
+To use response caching in the GraphOS Router, you must set up:
+
+- A Redis instance or cluster that your router instances can communicate with
+- A [GraphOS Enterprise plan](https://www.apollographql.com/pricing/) that [connects your router to GraphOS](/router/configuration/overview/#environment-variables).
```

Comment:
Available on all plans so we should remove this requirement. Maybe add caveat that rate limits and performance pricing applies